lib/vquic/ngtcp2.c

```diff
@@ -23,6 +23,7 @@
 #include "curl_setup.h"
 
 #ifdef USE_NGTCP2
+#include <assert.h>
```

Comment:
`ngtcp2.c` includes `curl_setup.h` and there on line 337: 

https://github.com/curl/curl/blob/f41deddde879c15350df880b1c8959bb521cd679/lib/curl_setup.h#L337

Can you figure out why that isn't enough?

(Also, we should probably convert the `assert()` calls to `DEBUGASSERT()`  to unify with other code.)

Reply:
Oh. that header looks like a mistake indeed.